core/indexer/processTransactions_test.go

```diff
@@ -271,6 +272,62 @@ func TestSetTransactionSearchOrder(t *testing.T) {
 	assert.True(t, txPoolHasSearchOrder(transactions, 1))
 }
 
+func TestGetGasUsedFromReceipt_RefundedGas(t *testing.T) {
+	t.Parallel()
+
+	gasPrice := uint64(1000)
+	gasLimit := uint64(10000)
+	recValue := big.NewInt(10000)
+	txHash := []byte("tx-hash")
+	rec := &receipt.Receipt{
+		Value:   recValue,
+		SndAddr: nil,
+		Data:    []byte(transactionProcess.RefundGasMessage),
+		TxHash:  txHash,
+	}
+	tx := &Transaction{
+		Hash: hex.EncodeToString(txHash),
+
+		GasPrice: gasPrice,
+		GasLimit: gasLimit,
+	}
+
+	expectedGasUsed := big.NewInt(0).SetUint64(gasPrice)
```

Comment:
`expectedGasUsed` as a constant / a big int initialized with the expected value?

Reply:
done.